manticore/manticore.py

```diff
@@ -469,20 +497,24 @@ def apply_model_hooks(self, path):
         # event code is in place.
         import core.cpu
         import importlib
-        import platforms
+        import models
 
         with open(path, 'r') as fnames:
             for line in fnames.readlines():
                 address, cc_name, name = line.strip().split(' ')
-                fmodel = platforms
+                cc = getattr(core.cpu.x86.ABI, cc_name)
```

Comment:
A lot of the code here seems reverted to much older code, including naming above (`platform` back to `model`)